config/main

```diff
@@ -0,0 +1 @@
+include config/default
```

Comment:
This is the better way I found to get fully working configuration that works development and tests. Like this we can have common docker compose file, that do not appear changed with git status.